app/(service)/recruit-detail/[shopId]/[recruitId]/_components/recruitCard.tsx

```diff
@@ -0,0 +1,104 @@
+'use client';
+
+import { useEffect, useState } from 'react';
+
+import Image from 'next/image';
+import { useParams } from 'next/navigation';
+
+import Button from '@/components/button/button';
+import PercentageBadge from '@/components/post/percentageBadge';
+import apiClient from '@/libs/axios';
+import calculatePercentage from '@/utils/calculatePercentage';
+import formatDateRange from '@/utils/formatDateRange';
+
+interface FetchData {
+  hourlyPay: 'number';
+  startsAt: 'string';
+  workhour: 'number';
```

Comment:
P3: `h`를 대문자로 바꿔주시면 좋을 것 같습니당

```suggestion
  workHour: 'number';
```

Reply:
코드잇에서 workhour 이렇게 보내줘서 소문자로 해놨었습니다

Reply:
헉...